hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFileSystem.java

```diff
@@ -135,6 +137,14 @@ public boolean hasPathCapability(final Path path, final String capability)
     if (cap) {
       return cap;
     }
+    // this switch is for features which are in the OFS client.
+    switch (validatePathCapabilityArgs(path, capability)) {
+    case CommonPathCapabilities.LEASE_RECOVERABLE:
```

Comment:
is there a reason not adding this switch case to OzonePathCapabilities.hasPathCapability() instead?

Reply:
Done.